lib/components/RecentBlocks.js

```diff
@@ -33,25 +33,28 @@ class RecentBlocks extends React.Component {
     // use this to ensure order of columns
     // otherwise table just uses object keys
     const colHeaders = [
```

Comment:
This array should only be created one time instead of in every render call, since it doesn't change

Reply:
good call. updated and did the same with mainData.